README.adoc

```diff
@@ -0,0 +1,51 @@
+:ext-relative:
+:toc: macro
+:toclevels: 4
+
+# DOME Examples
+
+Examples made with the https://domeengine.com[DOME engine]
+
+Make sure DOME is installed, then you can point it at each example directory.
```

Comment:
You should point out that git doesn't actually pull the submodules immediately, so you need to enter the right commands to make that happen. (You should find out what those are.)